src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -531,6 +531,25 @@ function renderSubtreeIntoContainer(
   );
 
   if (__DEV__) {
+    if (container._reactRootContainer && container.nodeType !== COMMENT_NODE) {
+      const hostInstance = DOMRenderer.findHostInstance(
+        container._reactRootContainer.current,
+      );
```

Comment:
If it would be possible to write such a method, without adding a flag to the `portalContainer` node, then there must already be something we could look at here that would tell us if it's a portal. Is that right?